tests/components/motionmount/test_sensor.py

```diff
@@ -0,0 +1,139 @@
+"""Tests for the MotionMount Sensor platform."""
+
+from unittest.mock import MagicMock, PropertyMock
+
+import motionmount
+
+from homeassistant.core import HomeAssistant
+
+from . import ZEROCONF_NAME
+
+from tests.common import MockConfigEntry
+
+MAC = bytes.fromhex("c4dd57f8a55f")
+
+
+async def test_error_status_sensor_none(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_motionmount_config_flow: MagicMock,
+) -> None:
+    """Tests the state attributes."""
+    mock_config_entry.add_to_hass(hass)
+
+    type(mock_motionmount_config_flow).name = PropertyMock(return_value=ZEROCONF_NAME)
+    type(mock_motionmount_config_flow).mac = PropertyMock(return_value=MAC)
+    type(mock_motionmount_config_flow).is_authenticated = PropertyMock(
+        return_value=True
+    )
+    assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
+
+    assert hass.states.get("sensor.my_motionmount_error_status").state == "none"
+
+
+async def test_error_status_sensor_motor(
```

Comment:
It appears the only thing that differs from these tests is the value of `system_status`. You can use the parametrize feature to do this all from one test.

You'd add something similar to:
`@pytest.mark.parametrize("system_status", [None, MotionMountSystemError.MotorError, ...])` and then add a parameter to the test `system_status: MotionMountSystemError | None` and assign that to the mocked property.